packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -2258,34 +2291,40 @@ class InputDecoration {
 
   @override
   int get hashCode {
+    // split into multiple hashValues calls
+    // because the hashValues function only takes up to 20 fields...
     return hashValues(
```

Comment:
This statement can be left as it is - two calls to hashValues(). Just more a few of the parameters to the second call up into the first parameter list.